src/braket/ahs/driving_field.py

```diff
@@ -83,6 +83,27 @@ def detuning(self) -> Field:
         r"""Field: global detuning (:math:`\Delta(t)`). Time is in s, and value is in rad/s."""
         return self._detuning
 
+    def stitch(self, other: DrivingField, boundary: str = "mean") -> DrivingField:
+        """Stitches two driving fields based on TimeSeries.stitch method.
+        Shifts time points in the second DrivingField to align with the first DrivingField.
+        Args:
```

Comment:
Lack of spacing might be interfering with docs rendering correctly; this applies to the other methods and files too
```suggestion
        Shifts time points in the second DrivingField to align with the first DrivingField.

        Args:
```

Reply:
ok, fixed